torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -529,6 +518,10 @@ int64_t KinetoEvent::cudaElapsedUs() const {
 }
 
 c10::DeviceType KinetoEvent::deviceType() const {
+  if (device_type_ >= 0) {
```

Comment:
Why make this `int8_t` and defaulted `-1`? Can you add invalid device type? Or this is tied, 1:1, to device types on aten side so we have to follow that?